src/azure-openapi-validator/rules/ControlCharactersAreNotAllowed.ts

```diff
@@ -0,0 +1,32 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import { MergeStates, OpenApiTypes, rules } from '../rule';
+export const ControlCharactersAreNotAllowed: string = "ControlCharactersAreNotAllowed";
+
+rules.push({
+  id: "SE40",
```

Comment:
what would happen if we miss one of the properties during the implementation of another rule? are there defaults?